specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/examples/CreateSuppression.json

```diff
@@ -21,6 +21,12 @@
           "ttl": "07:00:00:00"
         }
       }
+    },
+    "404": {
+      "body": {
+        "code": "NotFound",
+        "message": "Recommendation to be suppressed is not found. Subscription Id:{0} Recommendation Id:{1}"
```

Comment:
this is incorrect. Needs to have error as key.

Reply:
you may need to fix example in other suppression .

Reply:
Thanks for the catch! I've updated both of them.

Reply:
Any reason why schema validation is failing?

Reply:
Just realize that should be lowercase for A"rm"ErrorResponse for the model validation it is complaining no response body for CreateSuppression.json and GetSuppressionDetail.json which is weird because we are using ArmErrorResponse it should be "error" and not "body" like your comment has mentioned I will check how others are using it if that is identical with others' cases I will file for exception for false positive case.